### DISCUSSION TITLE
add instafade / circle hit expand options

### DISCUSSION THREAD

**Discussion by rikiaosu** [ppy/osu#25473] [Ideas] [answered]
![instafade](https://github.com/ppy/osu/assets/150896202/84fcd397-05e1-4b22-bca0-1fc65d4cf167)
hopefully this isn't too difficult to add, but if you could add a option to adjust the characteristics of circles expanding, it would be much appreciated.

currently, the only way to have an "instant fade" effect is to tie the circle to the default number, but being able to adjust it for the entire hitcircle would be nice.

i propose adding 3 sliders in settings to adjust this
1. duration of expansion when click (longer -> takes more time to do full animation, shorter -> less time , shortest value = instant disappear)
2. size of expansion (bigger -> circle expands more, similar to old osu circle expanding, see https://www.youtube.com/watch?v=03nFcUiR7Bg for example)
3. opacity of expansion (i.e. having the circle instantly be less opacity when hit but still have the animation can help with readability)

thank you for consideration :)

btw, i know there are like 2 other posts about instafade, but my suggestion is to add options that affect the entire hitcircle

**Comment by peppy** [comment, marked as answer]
Skin configuration will improve as time continues. For future reference, if you find a related post, post in it, don't create a new one (it's just noisy to us and we'll likely mute it).

**Reply by rikiaosu** [reply]
sorry, pishi told me to make a github post when i brought him the idea. ill keep that in mind for future reference
